src/js/main.js

```diff
@@ -10,17 +10,41 @@
 		obj[key] = src;
 	}
 
-	GameContext = {
+	function displayErrors(elem, errors) {
+		msg = ""
+
+		while (errors && errors.length > 0) {
+			error = errors.pop();
+			msg += (error + "<br>");
+		}
+		
+		elem.innerHTML = msg;
+		elem.display = msg.length > 0 ? elem.display : "none";
+	}
+
+	var errorContent = document.getElementById(errorContentId);

```

Comment:
I don't like the way this is being done.  Should probably use jQuery or similar.